_unittest/test_02_2D_modeler.py

```diff
@@ -1,15 +1,11 @@
 # standard imports
 import math
 
-try:
-    import pytest
-except ImportError:
-    import _unittest_ironpython.conf_unittest as pytest
```

Comment:
I think we need this for ironpython compatibility.

Reply:
Good point. Because of this, checking unused module imports with flake8 can't be applied to the unit test files, so I will probably just abandon this PR.